src/content/chainlink-automation/guides/streams-lookup-error-handler.mdx

```diff
@@ -0,0 +1,233 @@
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>No error</td>
+      <td>N/A</td>
+      <td>No error</td>
+      <td>200 (Status OK)</td>
+    </tr>
+    <tr>
+      <td rowspan="3">ErrCodeStreamsBadRequest: 800400</td>
+      <td rowspan="3">No</td>
+      <td>User requested 0 feeds</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>User error, incorrect parameter input</td>
+      <td>400 (StatusBadRequest)</td>
+    </tr>
+    <tr>
+      <td>Issue with encoding http url (bad characters)</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <td>ErrCodeStreamsUnauthorized: 808401</td>
+      <td>No</td>
+      <td>Key access issue or incorrect feedID</td>
+      <td>401 (StatusUnauthorized)</td>
+    </tr>
+    <tr>
+      <td>808206</td>
+      <td>Log trigger - after retries; Conditional immediately</td>
+      <td>Requested m reports but only received n (partial)</td>
+      <td>200 (Status OK) or StatusPartialContent</td>
+    </tr>
+    <tr>
+      <td>8085XX (e.g 808500)</td>
+      <td>Log trigger - after retries; Conditional immediately</td>
+      <td>No response</td>
+      <td>
+        <ul>
+          <li>500 (StatusInternalServerError)</li>
+          <li>502 (StatusBadGateway)</li>
+          <li>503 (StatusServiceUnavailable)</li>
+          <li>504 (StatusGatewayTimeout)</li>
+        </ul>
+      </td>
+    </tr>
+    <tr id="808600">
+      <td>ErrCodeStreamsBadResponse: 808600</td>
+      <td>No</td>
+      <td>Error in reading body of returned response, but service is up</td>
+      <td>200 but can't decode Data Streams output</td>
+    </tr>
+    <tr id="808601">
+      <td>ErrCodeStreamsTimeout: 808601</td>
+      <td>No</td>
+      <td>No valid report is received for 10 seconds (e.g. Data Streams returns 200 but no report)</td>
+      <td>No valid Data Streams report received within 10 seconds (200 but no report)</td>
+    </tr>
+    <tr id="808700">
+      <td>ErrCodeStreamsUnknownError: 808700</td>
+      <td>No</td>
+      <td>Unknown</td>
+      <td>Anything else</td>
+    </tr>
+  </tbody>
+</table>
+
+## Example code
+
+This example code highlights the `revert StreamsLookup`, `checkCallback`, `checkErrorHandler` and `performUpkeep` functions. The full code example is available [here](https://github.com/smartcontractkit/documentation/blob/main/public/samples/Automation/StreamsWithError.sol).
+
+```solidity
+...
+function checkLog(Log calldata log, bytes memory)
+    external
+    returns (bool upkeepNeeded, bytes memory performData)
+{
+    revert StreamsLookup(
+        STRING_DATASTREAMS_FEEDLABEL,
+        feedIds,
+        STRING_DATASTREAMS_QUERYLABEL,
+        log.timestamp,
+        ""
+    );
+}
+
+function checkCallback(bytes[] calldata values, bytes calldata extraData)
+    external
+    pure
+    returns (bool, bytes memory)
+{
+    bool success = true;
+    return (true, abi.encode(values, extraData, success));
+}
+
+function checkErrorHandler(
+    bytes calldata errorCode,
```

Comment:
Is `errorCode` of type `bytes calldata` or `uint`? In the snippet above, it is a `uint`.